v2/cmd/wails/internal/commands/dev/dev.go

```diff
@@ -438,12 +450,32 @@ func runFrontendDevWatcherCommand(cwd string, devCommand string) (func(), error)
 		wg.Done()
 	}()
 
+	var viteServerURL string
+	if discoverViteServerURL {
+		buf := bufio.NewReader(stdo)
+		for {
+			line, _, _ := buf.ReadLine()
+			match := bytes.Index(line, []byte("Local:"))
+			if match != -1 {
+				line := strings.TrimSpace(string(line[match+6:]))
+				viteServerURL = stripansi.Strip(line)
+				LogGreen("Vite Server URL: %s", viteServerURL)
+				_, err := url.Parse(viteServerURL)
+				if err != nil {
+					LogRed(err.Error())
+					break
+				}
+				break
+			}
+		}
+	}
+
```

Comment:
Is it fine to not fully consume the stdout pipe? IIRC on linux a command might block when the stdout's buffer is full, but I'm not sure if this is a problem when using stdout pipe with Go.